sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuWindowExec.scala

```diff
@@ -1227,10 +1231,13 @@ trait BasicWindowCalc extends Arm {
             cb.column(inputIndex).asInstanceOf[GpuColumnVector].getBase.incRefCount()
       }
 
-      withResource(GpuProjectExec.project(cb, initialProjections)) { initProjCb =>
-        aggregations.doAggs(isRunningBatched, boundOrderSpec, orderByPositions,
-          partByPositions, initProjCb, outputColumns)
-      }
+      val inputSpillable = SpillableColumnarBatch(
```

Comment:
Who closes inputSpillable?

Reply:
The retry code does close this. `withRetryNoSplit` for a single item like this should close the input after the function body was invoked, or after there is an exception we couldn't handle.